src/containers/GenerateToken/UserInfo.tsx

```diff
@@ -16,7 +16,7 @@ const UserInfo: React.FC = ({
 }) => {
   const pbKeyRef = useRef()
   const scKeyRef = useRef()
-  const publicKeyUrl = `https://proofof.dog/addr?dogecoin=${publicKey}`
+  const publicKeyUrl = `https://proofof.dog/addr?dogname=${dogname}&dogecoin=${publicKey}`
```

Comment:
reorder and ensure data are clean. 

- dogecoin=
- twitter= (lowercase, whitespace trimmed, @ symbol trimmed)
- dogname= (whitespace trimmed and truncated to max length 64)

Reply:
trim twitter to max length 64 also